tests/Requests/PostSubscriptionRequestTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\NotificationClient\Tests\Requests;
+
+use Keboola\NotificationClient\Requests\PostSubscription\EmailRecipient;
+use Keboola\NotificationClient\Requests\PostSubscription\Filter;
+use Keboola\NotificationClient\Requests\PostSubscriptionRequest;
+use PHPUnit\Framework\TestCase;
+
+class PostSubscriptionRequestTest extends TestCase
+{
+    public function testCreate(): void
+    {
+        $subscriptionRequest = new PostSubscriptionRequest(
+            'job_failed',
+            new EmailRecipient('john.doe@example.com'),
+            [new Filter('foo', 'bar')]
```

Comment:
Mozna bych tu pridal jeden test na situaci, kdy `$filters` nejsou pole indexovany od `0` (nebo to ma nedej boze custom klice). Casto se to pri encodovani do JSON zapomina osetrit a konci to tim, ze se to exportuje jako JSON objekt misto pole.

Reply:
dobrej nápad, done